engine/src/flutter/impeller/renderer/backend/vulkan/command_queue_vk.h

```diff
@@ -17,9 +17,9 @@ class CommandQueueVK : public CommandQueue {
 
   ~CommandQueueVK() override;
 
-  fml::Status Submit(
-      const std::vector<std::shared_ptr<CommandBuffer>>& buffers,
-      const CompletionCallback& completion_callback = {}) override;
+  fml::Status Submit(const std::vector<std::shared_ptr<CommandBuffer>>& buffers,
+                     const CompletionCallback& completion_callback = {},
+                     bool block_on_schedule = false) override;
```

Comment:
Default arguments are banned on virtual methods.

https://google.github.io/styleguide/cppguide.html#Default_Arguments

Reply:
Banned is maybe a heavy word here, discouraged.  I changed my mind, this is probably fine since it is a trivial type.